webapp-client/src/component/ErrorMessage.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default class ErrorMessage extends React.Component {
```

Comment:
There's a merge conflict with the master branch. Please rebase this to the tip of master and add a commit for resolving.

Reply:
Done